packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -157,11 +166,50 @@ export class ChoiceGroupOptionBase extends BaseComponent<IChoiceGroupOptionProps
     );
   }
 
+  private _onLabelWrapperRef(element: HTMLDivElement | null): void {
+    if (element) {
+      const labelWrapperElement = createRef<HTMLDivElement>();
+      labelWrapperElement.current = element;
+      this._labelWrapperElementList.push(labelWrapperElement.current);
```

Comment:
hmm. This looks sketchy. I don't see a pop. So, if the ref evals more than once, won't you just keep pushing?

Reply:
Good point. Where would you expect to see a pop?